File: Java/Petshop.java
class Petshop {
    private String ID;
    private String namaProduk;
    private String kategori;
    private int harga;

    public Petshop() {
        this.ID = "";
        this.namaProduk = "";
        this.kategori = "";
        this.harga = 0;
    }

    public void setData(String ID, String namaProduk, String kategori, int harga) {
        this.ID = ID;
        this.namaProduk = namaProduk;
        this.kategori = kategori;
        this.harga = harga;
    }

    public void reset() {
        this.ID = "";
        this.namaProduk = "";
        this.kategori = "";
        this.harga = 0;
    }

    public String getID() { return ID; }
    public String getNamaProduk() { return namaProduk; }
    public String getKategori() { return kategori; }
    public int getHarga() { return harga; }
}

File: PHP/Petshop.php
<?php
    // Memulai sesi untuk menyimpan data sementara
    session_start();
    if (!isset($_SESSION['data_petshop'])) {
        $_SESSION['data_petshop'] = [];
    }

    // Kelas Petshop untuk merepresentasikan produk
    class Petshop {
        private $ID;
        private $namaProduk;
        private $kategori;
        private $harga;
        private $gambar;

        // Konstruktor untuk menginisialisasi atribut
        public function __construct($ID, $namaProduk, $kategori, $harga, $gambar) {
            $this->ID = $ID;
            $this->namaProduk = $namaProduk;
            $this->kategori = $kategori;
            $this->harga = $harga;
            $this->gambar = $gambar;
        }

        // Getter untuk mengambil nilai atribut
        public function getID() {
            return $this->ID;
        }

        public function getNamaProduk() {
            return $this->namaProduk;
        }

        public function getKategori() {
            return $this->kategori;
        }

        public function getHarga() {
            return $this->harga;
        }

        public function getGambar() {
            return $this->gambar;
        }
    }

?>
